### ISSUE TITLE
Problem with EE2 and MSM

### ISSUE THREAD

**Issue by elena87** [bjornbjorn/SEO-Lite#32]
Hi, I'm running a site with EE 2.10.2 and SEO Lite 1.4.9.3.
In the main site it works well, but in the second site no SEO appears on the pages (no meta title, description and keywords). If I try to edit the default configuration in Modules - SEO Lite in the second site, it redirects to the front-end and it doesn't save them. What can I do?

**Comment by bjornbjorn** [comment]
Hi, in admin on your second site, please navigate to the Addons -> SEO Lite settings page and save settings for the second site.

Does that fix the issue?

**Comment by elena87** [comment]
No, it's when I click update that the browser shows me the front-end. 
Il 08 nov 2016 7:21 PM, Bjørn Børresen <notifications@github.com> ha scritto:Hi, in admin on your second site, please navigate to the Addons -> SEO Lite settings page and save settings for the second site.

Does that fix the issue?

**Comment by bjornbjorn** [comment]
Hi, I just tested this with EE 2.11.2, MSM and SEO Lite 1.4.9 and settings save on the second site w/o problems. So, I'm not able to reproduce. Maybe you could try upgrading EE to the latest 2-version and see if that fixes it.

**Comment by elena87** [comment]
Hi. In the meantime I create another site (the one made with EE2 is still there, with the problem) with EE 3.4.4, MSM with one additional site, and SEO Lite 1.5.4.
Back-end: in the setting tab, if I click update, the page refreshes and shows the front-end, and the settings are not updated (both in the main site and in the second site).
Front-end: in the main site SEO Lite works well (except for the home page, because I can't modify the default settings). In the second site it shows these errors (screenshot):
 

 
-----Messaggio originale-----Da: "Bjørn Børresen" <notifications@github.com>Inviato: Friday, 11 November, 2016 12:47A: "bjornbjorn/SEO-Lite" <SEO-Lite@noreply.github.com>Cc: "elena87" <elena.mascolo@nuvoleink.com>, "Author" <author@noreply.github.com>Oggetto: Re: [bjornbjorn/SEO-Lite] Problem with EE2 and MSM (#32)

Hi, I just tested this with EE 2.11.2, MSM and SEO Lite 1.4.9 and settings save on the second site w/o problems. So, I'm not able to reproduce. Maybe you could try upgrading EE to the latest 2-version and see if that fixes it.

**Comment by bjornbjorn** [comment]
>  In the second site it shows these errors (screenshot):

There's no screenshot attached,  could you please try posting it again?

**Comment by elena87** [comment]
I attached it again. If you can't see it, I uploaded temporarily here: http://elenina.it/seo-lite/SEO-Lite-MSM.png
 
-----Messaggio originale-----Da: "Bjørn Børresen" <notifications@github.com>Inviato: Wednesday, 23 November, 2016 12:55A: "bjornbjorn/SEO-Lite" <SEO-Lite@noreply.github.com>Cc: "elena87" <elena.mascolo@nuvoleink.com>, "Author" <author@noreply.github.com>Oggetto: Re: [bjornbjorn/SEO-Lite] Problem with EE2 and MSM (#32)


In the second site it shows these errors (screenshot):

There's no screenshot attached, could you please try posting it again?

**Comment by bjornbjorn** [comment]
Thanks! That frontend error is a result of the lack of settings on the second site.

I'm still not able to reproduce. Do you have any other addons installed and are you able to save settings in these?

Also, are you accessing the CP via admin.php or /system ?

**Comment by elena87** [comment]
It's a fresh install, the only other add-on installed is wygwam. I tried to change some settings and it had no problems.
 
I am accessing the CP through /system.
 
-----Messaggio originale-----
Da: "Bjørn Børresen" <notifications@github.com>
Inviato: Wednesday, 23 November, 2016 13:12
A: "bjornbjorn/SEO-Lite" <SEO-Lite@noreply.github.com>
Cc: "elena87" <elena.mascolo@nuvoleink.com>, "Author" <author@noreply.github.com>
Oggetto: Re: [bjornbjorn/SEO-Lite] Problem with EE2 and MSM (#32)



Thanks! That frontend error is a result of the lack of settings on the second site.
I'm still not able to reproduce. Do you have any other addons installed and are you able to save settings in these?
Also, are you accessing the CP via admin.php or /system ?
